builtin/mainmenu/dlg_create_world.lua

```diff
@@ -285,14 +296,17 @@ local function create_world_formspec(dialogdata)
 
 	end
 
-	retval = retval ..
-		"label[0,2;" .. fgettext("Mapgen") .. "]"..
-		"dropdown[0,2.5;6.3;dd_mapgen;" .. mglist .. ";" .. selindex .. "]"
+	if mapgens_descriptions[current_mg] then
+		retval = retval ..
+			"label[0,2;" .. fgettext("Mapgen") .. "]"..
+			"dropdown[0,2.5;6.3;dd_mapgen;" .. mglist .. ";" .. selindex .. "]"..
+			"textarea[0.5,3.2;6,2;;;" .. mapgens_descriptions[current_mg] .. "]"
```

Comment:
so the dropdown will disappear if for some reason there is no description, why?

Reply:
^ this is still unresolved as far as i can tell?